lib/standard_deviation_module.rb

```diff
@@ -0,0 +1,29 @@
+module StandardDeviation
+
+  def standard_deviation_age
```

Comment:
I would rename this method standard_deviation, since this module doesn't have anything to do with ages. That's in the Event file. In the event file, you could have something like this:

```ruby
def standard_deviation_age
  standard_deviation(@ages)
end
```

Reply:
Got it.